third_party/istio-1.1-latest

```diff
@@ -1 +1 @@
-istio-1.1.3
\ No newline at end of file
```

Comment:
<!--whitespace-bot-->
```suggestion
istio-1.1.7

```

Reply:
We cannot add an empty line at the end, since it is a symbolic link instead of a file.
If we force to add an empty line, the link will be broken. @mattmoor-sockpuppet